askama/src/lib.rs

```diff
@@ -478,6 +488,36 @@ pub mod actix_web {
     }
 }
 
+#[cfg(feature = "with-gotham")]
+pub mod gotham {
+    extern crate gotham;
+    extern crate hyper;
+    extern crate mime_guess;
```

Comment:
@djc In Rust 2018 we should not eliminate these `extern crate`?

Reply:
I've removed them in the latest commit